src/shadowbox/server/manager_service.ts

```diff
@@ -28,7 +28,7 @@ import {ServerConfigJson} from './server_config';
 import {SharedMetricsPublisher} from './shared_metrics';
 
 // Creates a AccessKey response.
-function accessKeyToJson(accessKey: AccessKey) {
+function accessKeyResponseJson(accessKey: AccessKey) {
```

Comment:
nit: I think the previous name was clearer as it implied an action: convert/serialize.

Reply:
Done on  c9ae52d